.travis.yml

```diff
@@ -18,16 +18,28 @@ matrix:
     - os: linux
       dist: xenial
       compiler: clang
+    - os: linux
+      dist: bionic
+      compiler: gcc
+    - os: linux
+      dist: bionic
+      compiler: clang
     - os: osx
       osx_image: xcode9.4
     - os: osx
       osx_image: xcode10
     - os: osx
       osx_image: xcode10.1
+    - os: osx
+      osx_image: xcode10.2
+    - os: osx
+      osx_image: xcode10.3
+    - os: osx
+      osx_image: xcode11
 sudo: false
 language: cpp
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install re2c             ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then HOMEBREW_NO_AUTO_UPDATE=1 HOMEBREW_NO_INSTALL_CLEANUP=1 brew install re2c ; fi
```

Comment:
Note: TravisCI does have a specific syntax for installing prereqs that's supposed to behave "optimally" with homebrew.

It doesn't work, and the error output that it gives does not hint at any way to resolve the problem.

So while I'd rather use the dedicated syntax, this is what works, so it's what I'm proposing to be merged.

Reply:
I see, thanks for the info.